pkgs/development/python-modules/bilibili-api-python/default.nix

```diff
@@ -37,6 +37,7 @@ buildPythonPackage rec {
   pythonRelaxDeps = [
     "beautifulsoup4"
     "lxml"
+    "pillow"
```

Comment:
I am surprised this is necessary; it builds successfully for me without this new relaxDep